tests/rptest/tests/metrics_reporter_test.py

```diff
@@ -114,6 +117,13 @@ def assert_fields_are_the_same(metadata, field):
         assert_fields_are_the_same(metadata, 'cluster_uuid')
         assert_fields_are_the_same(metadata, 'cluster_created_ts')
 
+        # Check config values
+        assert last["config"]["retention_bytes"] == "[value]"
```

Comment:
I'm not sure I understand the intent: isn't retention_bytes something that we should be collecting, as it's numeric + can't contain secrets?

Reply:
We do not wanna hardcode config option which we wanted to report. This info from metrics_reporter should only signal about enabled features. I am not sure about how sensitive info about timouts and etc. But we decided do not provide it